mne/viz/_3d.py

```diff
@@ -415,6 +454,19 @@ def plot_source_estimates(stc, subject=None, surface='inflated', hemi='lh',
         View to use. See surfer.Brain().
     colorbar : bool
         If True, display colorbar on scene.
+    limits : str | 3-tuple | dict
```

Comment:
do we really need all these options? Couldn't we do everything using a `dict`? To me, it would be super-confusing to a new user if not documented properly.